updating poses for autos

grace-mills opened the pull request.

grace-mills committed:
Merge remote-tracking branch 'origin/main' into gmills

grace-mills committed:
current paths

jmullins16 committed:
Updated centerline poses

frc2135 committed:
new tuner constants file

jmullins16 reviewed (changes requested):
Something went haywire here. The code implies that we changed the CAN ID definitions, and I don't think that's what you did. I thought you moved the motors and CANcoders to use our original CAN ID naming (which is consistent across both robots).

Now resolved.

frc2135 committed:
Create new Tuner X projects after swerve module swap

Obtained new offsets and tested values

frc2135 committed:
Removed some duplicate files that don't belong

navgrid was left over from someone running PathPlanner in the wrong folder, and tuner-swerve-project is a duplicate of tuner-project in the same directory.

jmullins16 committed:
Merge Tuner X constants into Comp2024 shared TunerConstants file for both Comp and Practice robots

jmullins16 reviewed (commented):
Please review updated changes fixed to maintain the CAN IDs that we had before

jmullins16 reviewed (approved):
Good to go now.
